packages/wonder-blocks-tooltip/components/tooltip.js

```diff
@@ -175,20 +158,27 @@ export default class Tooltip extends React.Component<Props, State> {
     }
 
     _renderTooltipAnchor(ids?: IIdentifierFactory) {
-        const {forceAnchorFocusivity} = this.props;
+        const {children, forceAnchorFocusivity} = this.props;
+        const {active, anchorElement} = this.state;
+
+        const popperHost =
+            maybeGetPortalMountedModalHostElement(anchorElement) ||
+            document.querySelector("body");
+
         return (
-            <TooltipAnchor
-                forceAnchorFocusivity={forceAnchorFocusivity}
-                anchorRef={(r) => this._updateAnchorElement(r)}
-            >
-                {(active) => (
-                    <TooltipPortalMounter
-                        anchor={this._renderAnchorElement(ids)}
-                    >
-                        {this._renderPopper(active, ids)}
-                    </TooltipPortalMounter>
-                )}
-            </TooltipAnchor>
+            <React.Fragment>
+                <TooltipAnchor
+                    forceAnchorFocusivity={forceAnchorFocusivity}
+                    anchorRef={(r) => this._updateAnchorElement(r)}
+                    onActiveChanged={(active) => this.setState({active})}
+                    ids={ids}
+                >
+                    {children}
+                </TooltipAnchor>
+                {popperHost &&
+                    active &&
+                    ReactDOM.createPortal(this._renderPopper(ids), popperHost)}
+            </React.Fragment>
```

Comment:
Check out the `react-popper` docs. I think that this could all be simplified by using their React 16-friendly syntax.

```
<Manager>
    <Reference>
       ...anchor here
    </Reference>
    <Popper>
    </Popper>
</Reference>
```

I think their docs even mention working with portals.

That said, perhaps for a different diff. This seems like it would work ok.